sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -252,6 +252,7 @@ class StreamExecution(
    */
   private def runBatches(): Unit = {
     try {
+      sparkSession.sparkContext.setJobGroup(runId.toString, getBatchDescriptionString)
```

Comment:
Had this set to false due to HDFS-1208, but setting it to true since the HDFS bug is 7 years old.